plugins/modules/podman_generate_systemd.py

```diff
@@ -0,0 +1,525 @@
+#!/usr/bin/python
+
+# 2022, Sébastien Gendre <seb@k-7.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: podman_generate_systemd
+author:
+  - Sébastien Gendre (@CyberFox001)
+short_description: Generate systemd unit from a pod or a container
+description:
+  - Generate systemd .service unit file(s) from a pod or a container
+  - Support Ansible check mode
+options:
+  name:
+    description:
+      - Name of the pod or container to export
+    type: str
+    required: true
+  dest:
+    description:
+      - Destination of the generated systemd unit file(s)
+    type: str
```

Comment:
`ERROR: plugins/modules/podman_generate_systemd.py:0:0: doc-type-does-not-match-spec: Argument 'dest' in argument_spec defines type as 'path' but documentation defines type as 'str'`

Reply:
Ah, I forget to update the documentation.

Done.